### ISSUE TITLE
No output displayed after pressing o

### ISSUE THREAD

**Issue by dehiker** [inducer/pudb#446]
I ran a simple script with following command:
**python -m pudb test.py**

Press 'o' when stopping at Line 4, nothing but hints "Hit Enter to return:" displayed, as shown in image below:
![pudb_o_err](https://user-images.githubusercontent.com/1631480/115652906-bd1e4e80-a360-11eb-9d53-0eaeb639a8f9.png)

Anything wrong with what I did? Thanks!

**Comment by inducer** [comment]
Behaves as expected for me:
![image](https://user-images.githubusercontent.com/352067/115721595-cfd65a80-a343-11eb-8d5b-426f9e0bfc7d.png)
and after I hit Enter:
![image](https://user-images.githubusercontent.com/352067/115721648-db298600-a343-11eb-98fe-686b1cf7c435.png)

**Comment by asmeurer** [comment]
Seems like it's probably a terminal issue. Try seeing if you get the same error in a new terminal tab. 

Assuming it does, see if changing the display driver in the pudb settings changes anything. It would also help to know what terminal emulator program you are using and your $TERM environment variable.

**Comment by asmeurer** [comment]
Also, this is more likely a bug in urwid than pudb, since it handles enabling and disabling the raw output. Make sure you have the latest urwid version installed.

**Comment by dehiker** [comment]
Thank you guys for your prompt reply! @inducer @asmeurer 

I tried all the options listed below, nothing changed.
- Display driver:
│( ) auto
│( ) raw
│( ) curses
- $TERM: xterm-256color/xterm
- urwid: 2.1.2, Requirement already satisfied

But It seems the SSH client matters. As I connected to the server within WSL in windows terminal, **o** does not work. But it does work as expected when connecting with xshell.

Any suggestions? Thank you all!
